server/config.src.yaml

```diff
@@ -1,6 +1,11 @@
 clientId: clientid
 clientSecret: clientsecret
+projectId: projectId
```

Comment:
are these dummy values? then why we need them

Reply:
Yes, these are just dummy values. `config.src.yaml` is the configuration source file where these values are [used when testing](https://aqueduct.io/docs/application/configure/#configuration-conventions-and-deployment-options), and it's [supposed to have the same structure](https://aqueduct.io/docs/tut/writing-tests/#configuration-template) as `config.yaml`.